Taskfile.yml

```diff
@@ -46,59 +46,101 @@ tasks:
         <(cd '{{.OUTPUT_DIR}}' && tar cf - {{.CHECKSUM_TAR_BASE_ARGS}} . | md5sum)
         "{{.CHECKSUM_FILE}}"
 
+  download-projects:
```

Comment:
Ah, this was the issue I was talking about in the new deployment [docs](https://github.com/y-scope/yscope-docs/blob/c02856bb60a0744d69b12457983ca65691edbcb0/docs/dev-guide/misc-deploying.md). The short version is that each project (and version) might have different dependencies, some of which might conflict with each other; so having a single environment that can build all the sites will be impractical in the future. Instead, (eventually) we can have each repo use a GH workflow to build its own docs in a container and then publish them directly to docs.yscope.com.

For users to build the complete docs.yscope.com site manually, it will take a bit of effort (I tried to automate the cloning part at least), but I imagine this will be rare as long as we automate publishing the docs.